pkg/ytconfig/node.go

```diff
@@ -323,11 +325,18 @@ func getDataNodeServerCarcass(spec *ytv1.DataNodesSpec) (DataNodeServer, error)
 		if quota != nil {
 			storeLocation.Quota = *quota
 
+			if location.LowWatermark != nil {
+				storeLocation.LowWatermark = location.LowWatermark.Value()
+			} else {
+				gb := float64(1024 * 1024 * 1024)
+				storeLocation.LowWatermark = int64(math.Min(0.1*float64(storeLocation.Quota), float64(25)*gb))
+			}
```

Comment:
We discussed it in DM, this looks as a reasonable default to me

IMO, all the further fine tuning should be done via dynamic config.